File: Problem771.cs
namespace LeetCode;

public class Problem771
{
    static int NumJewelsInStones(string jewels, string stones)
    {
        int req = 0;
        foreach (var s in stones)
        {
            foreach (var j in jewels)
            {
                if (j == s)
                {
                    req++;
                }
            }
        }
        return req;
    }
    
    static void Main(string[] args) {
        Console.WriteLine(NumJewelsInStones("a", "Aaaa"));
    }
}

File: Problem771.java
public class Problem771
{
    static int numJewelsInStones(String jewels, String stones)
    {
        int req = 0;
        for (char s : stones.toCharArray())
        {
            for (char j : jewels.toCharArray())
            {
                if (j == s)
                {
                    req++;
                }
            }
        }
        return req;
    }
    
    static void Main(String[] args) {
        System.out.println(numJewelsInStones("a", "Aaaa"));
    }
}